contrib/drivers/drivers.go

```diff
@@ -148,10 +158,9 @@ func Clean(d VolumeDriver) VolumeDriver {
 	default:
 		break
 	}
+
 	d.Unset()
```

Comment:
It's better to handle Unset error also. I have seen you have handled the error in Setup()

Reply:
as the old code's implementation, I think the unset error is ignorable, so I didn't change here.

Reply:
ok